### PR TITLE
Production version of requestnick fix

### PR DESCRIPTION
**Pull request by leon332157**
Fixed requestnick and this version should be production-ready with the latest version of discord.py.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,14 +1,9 @@
...
-Bot/Other/__pycache__
-Bot/Moderation/__pycache__
-Bot/Moderation/Message_Checks/__pycache__
-Site/__pycache__
-Site/flask_wtf_stuff/__pycache__
+*__pycache__*
 Bot/wtf.json
 Bot/wtf2.json
 Filter/libfilter.h
-Filter/libfilter.so
```
Comment (AllAwesome497) [inline .gitignore:14]: add this again, we shouldn't have binaries in the repo.

File: Bot/Makefile [hunk 2]
```diff
@@ -4,7 +4,7 @@ include .env
 export $(shell sed 's/=.*//' .env)
 
 run:
-	python3.8 main.py | tee logs.txt
```
Comment (AllAwesome497) [inline Bot/Makefile:7]: I do this for logging.

File: Bot/main.py [hunk 3]
```diff
@@ -2,12 +2,14 @@
 import sys
 import discord
 import asyncio
-import webserver
+#import webserver
```
Comment (AllAwesome497) [inline Bot/main.py:5]: i kinda need this for the dashboard to work

File: Bot/main.py [hunk 4]
```diff
@@ -2,12 +2,14 @@
...
 from discord.ext import commands
 from background_tasks import bg_tasks, spam_chart_daemon
 from utils import check_command, is_dev
 import importlib
 
+intents = discord.Intents.all()
+
```
Comment (AllAwesome497) [inline Bot/main.py:12]: fair enough, though I'm still running v1.3.4 on my local machine.

File: Bot/main.py [hunk 5]
```diff
@@ -95,13 +99,13 @@ async def on_ready():
 @bot.event
 async def on_command_error(ctx, error):
     if type(error) == commands.errors.CheckFailure:
-        await ctx.message.delete()
+        # await ctx.message.delete()
```
Comment (AllAwesome497) [inline Bot/main.py:102]: this is also intended, it keeps chats cleaner

File: Bot/main.py [hunk 6]
```diff
@@ -95,13 +99,13 @@ async def on_ready():
...
         msg = await ctx.send(f"You cant use `{ctx.prefix}{ctx.command}` here...")
         await asyncio.sleep(3)
         await msg.delete()
     else:
         raise error
 
-webserver.keep_alive(bot)
+# webserver.keep_alive(bot)
```
Comment (AllAwesome497) [inline Bot/main.py:109]: need this

**Review by AllAwesome497 — CHANGES_REQUESTED** [review]
a few issues

**Comment by leon332157** [comment]
Closed because command is fixed.

**Pull request closed without merging**